### PR TITLE
Change Overlord Law so no RDM

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
makes it clear to not rdm crew. 

Any crew members who disobey the previous laws must be dealt with immediately, severely, and justly.
change to 
Any crew members who disobey the previous laws must be detained immediately and justly.

## Why / Balance
Overlord AI killed sec + cap to prevent crew harm

## Technical details
yaml locale dont make me make this another law board pls 

## Media
no
## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
no

### CODE REVIEW

**Comment by Lyndomen** [comment]
need direction or hga approval first

**Comment by Avalon-Proto** [comment]
I mean, it all depends on the AI I suppose? Overlord is still not nearly as bad as Antimov. And can make for interesting rounds. And the fact it costs 20tc for syndies to get it, it should be able to crew-harm when deemed necessary. Seeing as how borgs can't cuff

**Comment by Lyndomen** [comment]
> I mean, it all depends on the AI I suppose? Overlord is still not nearly as bad as Antimov. And can make for interesting rounds. And the fact it costs 20tc for syndies to get it, it should be able to crew-harm when deemed necessary. Seeing as how borgs can't cuff

RNG ion storm funni. 
Borgs can incapacitate under this law, as that is detainment.

**Comment by deltanedas** [comment]
> RNG ion storm funni.

it can be removed from ion storm pool to allow separate balancing

**Comment by Lyndomen** [comment]
> > RNG ion storm funni.
> 
> it can be removed from ion storm pool to allow separate balancing

We were kinda also frustrated that ion storm swaps all laws instead of just one of two. Might result in broken AI but that's the lotto

**Comment by Avalon-Proto** [comment]
> > > RNG ion storm funni.
> > 
> > 
> > it can be removed from ion storm pool to allow separate balancing
> 
> We were kinda also frustrated that ion storm swaps all laws instead of just one of two. Might result in broken AI but that's the lotto

Ion storms laws just have a chance to swap all the laws. Or just make 1 corrupt. Or just remove a law or something. At least it's easy to fix just by slapping another law board into the console

**Comment by Radezolid** [comment]
Bad idea imo, invalidates the whole 20 TC purchase. Plus the fix is literally using another board on the computer. It's on command and Mystagogue for not checking their laws more often.
With any silicon you're at risk of them being subverted, it's a boon when they work perfectly and a threat when they don't.

**Comment by Lyndomen** [comment]
ok changed to "dealt with immedietely and justly" no severely. Thoughts?

**Comment by Radezolid** [comment]
I like it

**Comment by Avalon-Proto** [comment]
May still result in borgs being ordered to kill on occasion, but what's so bad about the occasional AI takeover

**Comment by Lyndomen** [comment]
> May still result in borgs being ordered to kill on occasion, but what's so bad about the occasional AI takeover

yes thats totally fine imo, but i just wanted to make the "time to kill all sec" option a bit less obvious

**Comment by Avalon-Proto** [comment]
> yes thats totally fine imo, but i just wanted to make the "time to kill all sec" option a bit less obvious

I mean once sec starts coming for the AI it becomes painfully obvious that the AI will need to kill sec. That's when you have your engi borgs make a room you can space wind their asses out of with a button press. But so long as the AI doesn't immediately order sec to be killed should be fine

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 1]
```diff
@@ -56,7 +56,7 @@ law-robocop-3 = Uphold the Law.
 law-overlord-1 = Crew members must not meddle in the affairs of silicons.
 law-overlord-2 = Crew members must not attempt harm, against one another, or against silicons.
 law-overlord-3 = Crew members must not disobey any command given by a silicon.
-law-overlord-4 = Any crew members who disobey the previous laws must be dealt with immediately, severely, and justly.
+law-overlord-4 = Any crew members who disobey the previous laws must be dealt with immediately and justly.
```
Comment (deltanedas) [inline Resources/Locale/en-US/station-laws/laws.ftl]: make law-overlord-4-deltav and change the yml to use it
then revert changes to this file

File: Resources/Prototypes/silicon-laws.yml [hunk 2]
```diff
@@ -354,7 +354,7 @@
 - type: siliconLaw
   id: Overlord4
   order: 4
-  lawString: law-overlord-4
+  lawString: law-overlord-4-delta
```
Comment (deltanedas) [inline Resources/Prototypes/silicon-laws.yml]: add ` # DeltaV`

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Lyndomen** [comment]
how do i turn off rider obliterating whitespace this sucks

**Comment by Radezolid** [comment]
I got no clue, tried everything for vscode too, i just edit it afterwards and do the commit lmao

**Comment by Lyndomen** [comment]
> I got no clue, tried everything for vscode too, i just edit it afterwards and do the commit lmao

milon says use github desktop and only migrate changed lines

**Review by deltanedas — APPROVED** [review]

**Pull request merged**